File: src/tool/spring-boot-generator/output/test/db/weldingInfo.js
let weldingInfo = {
  name: "weldingInfo",
  description: "焊缝信息",
  columns: {
    id: {
      name: "id",
      description: "焊缝编号",
      isPk: true,
      autoIncrement: true,
      nullable: false,
      type: {
        name: "int",
        length: null
      }
    },
    weldingNo: {
      name: "weldingNo",
      description: "焊缝标识",
      nullable: true,
      type: {
        name: "varchar",
        length: null
      }
    },
    workareaId: {
      name: "workareaId",
      description: "工区编号",
      nullable: true,
      type: {
        name: "int",
        length: null
      }
    },
    stationId: {
      name: "stationId",
      description: "车站编号",
      nullable: true,
      type: {
        name: "int",
        length: null
      }
    },
    thighTrackId: {
      name: "thighTrackId",
      description: "股道编号",
      nullable: true,
      type: {
        name: "int",
        length: null
      }
    },
    lineId: {
      name: "lineId",
      description: "线路编号",
      nullable: true,
      type: {
        name: "int",
        length: null
      }
    },
    miles: {
      name: "miles",
      description: "里程",
      nullable: true,
      type: {
        name: "float"
      }
    },
    side: {
      name: "side",
      description: "线别",
      nullable: true,
      type: {
        name: "int",
        length: null
      }
    },
    direction: {
      name: "direction",
      description: "行别",
      nullable: true,
      type: {
        name: "int",
        length: null
      }
    },
    turnoutId: {
      name: "turnoutId",
      description: "道岔号",
      nullable: true,
      type: {
        name: "int",
        length: null
      }
    },
    weldingType: {
      name: "weldingType",
      description: "焊缝类型",
      nullable: true,
      type: {
        name: "int",
        length: null
      }
    },
    weldingWay: {
      name: "weldingWay",
      description: "焊缝方式",
      nullable: true,
      type: {
        name: "int",
        length: null
      }
    },
    positionCategory: {
      name: "positionCategory",
      description: "焊缝位置类别",
      nullable: true,
      type: {
        name: "int",
        length: null
      }
    },
    trackId: {
      name: "trackId",
      description: "轨道编号",
      nullable: true,
      type: {
        name: "int",
        length: null
      }
    },
    thignTrackCategory: {
      name: "thignTrackCategory",
      description: "股别",
      nullable: true,
      type: {
        name: "int",
        length: null
      }
    },
    turnoutThignCategory: {
      name: "turnoutThignCategory",
      description: "道岔股别",
      nullable: true,
      type: {
        name: "int",
        length: null
      }
    },
    position: {
      name: "position",
      description: "道岔位置",
      nullable: true,
      type: {
        name: "int",
        length: null
      }
    },
    holeNum: {
      name: "holeNum",
      description: "带孔数量",
      nullable: true,
      type: {
        name: "int",
        length: null
      }
    },
    trackGlancingFlatness: {
      name: "trackGlancingFlatness",
      description: "轨面平直度",
      nullable: true,
      type: {
        name: "float"
      }
    },
    innerGlancingFlatness: {
      name: "innerGlancingFlatness",
      description: "内测平直度",
      nullable: true,
      type: {
        name: "float"
      }
    },
    weldTrackTemperature: {
      name: "weldTrackTemperature",
      description: "焊接时轨温",
      nullable: true,
      type: {
        name: "float"
      }
    },
    weldDate: {
      name: "weldDate",
      description: "焊接日期",
      nullable: true,
      type: {
        name: "date"
      }
    },
    reinforceWelding: {
      name: "reinforceWelding",
      description: "是否加固焊接",
      nullable: true,
      type: {
        name: "int",
        length: null
      }
    },
    weldUser: {
      name: "weldUser",
      description: "鐒婃帴浜?",
      nullable: true,
      type: {
        name: "varchar",
        length: null
      }
    },
    welderUnit: {
      name: "welderUnit",
      description: "焊接人员所属单位",
      nullable: true,
      type: {
        name: "varchar",
        length: null
      }
    },
    status: {
      name: "status",
      description: "状态",
      nullable: true,
      type: {
        name: "int",
        length: null
      }
    },
    isDelete: {
      name: "isDelete",
      description: "删除标记",
      nullable: false,
      type: {
        name: "int",
        length: null
      }
    },
    remark: {
      name: "remark",
      description: "备注",
      nullable: true,
      type: {
        name: "varchar",
        length: null
      }
    },
    createTime: {
      name: "createTime",
      description: "创建时间",
      nullable: true,
      type: {
        name: "datetime"
      }
    },
    updateTime: {
      name: "updateTime",
      description: "更新时间",
      nullable: true,
      type: {
        name: "datetime"
      }
    },
    operator: {
      name: "operator",
      description: "操作人",
      nullable: true,
      type: {
        name: "int",
        length: null
      }
    }
  }
};

let columnsArray = [
  "id",
  "weldingNo",
  "workareaId",
  "stationId",
  "thighTrackId",
  "lineId",
  "miles",
  "side",
  "direction",
  "turnoutId",
  "weldingType",
  "weldingWay",
  "positionCategory",
  "trackId",
  "thignTrackCategory",
  "turnoutThignCategory",
  "position",
  "holeNum",
  "trackGlancingFlatness",
  "innerGlancingFlatness",
  "weldTrackTemperature",
  "weldDate",
  "reinforceWelding",
  "weldUser",
  "welderUnit",
  "status",
  "isDelete",
  "remark",
  "createTime",
  "updateTime",
  "operator"
];

function create() {
  return {
  id: 441,
  weldingNo: "焊缝标识448",
  workareaId: 437,
  stationId: 108,
  thighTrackId: 364,
  lineId: 656,
  miles: 497,
  side: 177,
  direction: 327,
  turnoutId: 268,
  weldingType: 19,
  weldingWay: 285,
  positionCategory: 721,
  trackId: 260,
  thignTrackCategory: 412,
  turnoutThignCategory: 30,
  position: 580,
  holeNum: 407,
  trackGlancingFlatness: 270,
  innerGlancingFlatness: 651,
  weldTrackTemperature: 703,
  weldDate: "2019-12-23 1:3:35",
  reinforceWelding: 913,
  weldUser: "鐒婃帴浜?670",
  welderUnit: "焊接人员所属单位921",
  status: 856,
  isDelete: 684,
  remark: "备注48",
  createTime: "2019-12-23 1:3:35",
  updateTime: "2019-12-23 1:3:35",
  operator: 640
};
}

weldingInfo.columnsArray=columnsArray;
weldingInfo.create=create;
module.exports=weldingInfo;

File: src/tool/spring-boot-generator/output/test/test/src/main/java/com/test/pojo/entity/WeldingInfo.java
package com.test.pojo.entity;

import com.test.pojo.entity.WeldingInfo;
import io.swagger.annotations.ApiModel;
import io.swagger.annotations.ApiModelProperty;
import lombok.Getter;
import lombok.Setter;

import java.util.Date;
import javax.validation.constraints.NotNull;



@ApiModel("焊缝信息")
@Getter
@Setter
public class WeldingInfo @extends {

    @ApiModelProperty("创建时间")
    private Date createTime;

    @ApiModelProperty("行别")
    private Integer direction;

    @ApiModelProperty("带孔数量")
    private Integer holeNum;

    @ApiModelProperty("焊缝编号")
    private Integer id;

    @ApiModelProperty("内测平直度")
    private Float innerGlancingFlatness;

    @ApiModelProperty("删除标记")
    @NotNull
    private Integer isDelete;

    @ApiModelProperty("线路编号")
    @NotNull
    private Integer lineId;

    @ApiModelProperty("里程")
    @NotNull
    private Float miles;

    @ApiModelProperty("操作人")
    @NotNull
    private Integer operator;

    @ApiModelProperty("道岔位置")
    @NotNull
    private Integer position;

    @ApiModelProperty("焊缝位置类别")
    @NotNull
    private Integer positionCategory;

    @ApiModelProperty("是否加固焊接")
    @NotNull
    private Integer reinforceWelding;

    @ApiModelProperty("备注")
    @NotNull
    private String remark;

    @ApiModelProperty("线别")
    @NotNull
    private Integer side;

    @ApiModelProperty("车站编号")
    @NotNull
    private Integer stationId;

    @ApiModelProperty("状态")
    @NotNull
    private Integer status;

    @ApiModelProperty("股道编号")
    @NotNull
    private Integer thighTrackId;

    @ApiModelProperty("股别")
    @NotNull
    private Integer thignTrackCategory;

    @ApiModelProperty("轨面平直度")
    @NotNull
    private Float trackGlancingFlatness;

    @ApiModelProperty("轨道编号")
    @NotNull
    private Integer trackId;

    @ApiModelProperty("道岔号")
    @NotNull
    private Integer turnoutId;

    @ApiModelProperty("道岔股别")
    @NotNull
    private Integer turnoutThignCategory;

    @ApiModelProperty("更新时间")
    @NotNull
    private Date updateTime;

    @ApiModelProperty("焊接日期")
    @NotNull
    private Integer weldDate;

    @ApiModelProperty("焊接时轨温")
    @NotNull
    private Float weldTrackTemperature;

    @ApiModelProperty("鐒婃帴浜?")
    @NotNull
    private String weldUser;

    @ApiModelProperty("焊接人员所属单位")
    @NotNull
    private String welderUnit;

    @ApiModelProperty("焊缝标识")
    @NotNull
    private String weldingNo;

    @ApiModelProperty("焊缝类型")
    @NotNull
    private Integer weldingType;

    @ApiModelProperty("焊缝方式")
    @NotNull
    private Integer weldingWay;

    @ApiModelProperty("工区编号")
    @NotNull
    private Integer workareaId;

}